css/style.css

```diff
@@ -0,0 +1,77 @@
+#nav-bar {
+    min-width: 290px;
+    width: 100%;
+    height: 100px;
+    top: 0;
+    position: fixed;
+    text-align: center;
+    border-bottom: 1px solid black ;
+}
+
+.nav-link {
+padding: 20px;
+margin: 20px;
+font-size: medium;
+
+}
+
+.nav-link a {
+text-decoration: none;
+color: black;
+}
+
+.div-class {
+    margin-top: 120px;
+    position: relative;
+}
+
+.flex-container {
+    display: flex;
+    flex-wrap: nowrap;
+    background-color: grey;
+    width:min-content;
+}
+
+.flex-container > div {
+    background-color: #f1f1f1;
+    width: 300px;
+    margin: 10px;
+    text-align: center;
+    line-height: 75px;
+    font-size: 30px;
```

Comment:
I like the way you used flexbox in such an early stage. The issue to point out here is the colors and positioning. To make it more presentable you should use better colors so it's pleasant to the eye.
The width property is making your flexbox in the left when it can be on the center by adding the justify-content property. Also, make the flex-wrap: wrap so it's not vertical.